cmd/kib.go

```diff
@@ -182,7 +183,12 @@ func ExtractData(mainHTML *goquery.Selection) Card {
 		Ability:     ability,
 		Version:     CardModelVersion,
 		Cardcode:    complex,
-		ImageURL:    imageCardURL,
+	}
+	if fullURL, err := url.JoinPath(config.baseURL, imageCardURL); err == nil {
+		card.ImageURL = fullURL
```

Comment:
Why not, but we should increment the `Version`

Reply:
Ok.